src/main/java/io/reactivex/Completable.java

```diff
@@ -943,10 +943,9 @@ public final Completable delay(final long delay, final TimeUnit unit, final Sche
      * @throws NullPointerException if onComplete is null
      */
     @SchedulerSupport(SchedulerSupport.NONE)
-    public final Completable doOnComplete(Action onComplete) {
-        return doOnLifecycle(Functions.emptyConsumer(), Functions.emptyConsumer(),
-                onComplete, Functions.EMPTY_ACTION,
-                Functions.EMPTY_ACTION, Functions.EMPTY_ACTION);
+    public final Completable doOnComplete(final Action onComplete) {
```

Comment:
How is this any different from [this](https://github.com/ReactiveX/RxJava/pull/4483/files#diff-68a5181068fafd25f58f5c4600ac7bcaR1591)?